appsTransport_test.go

```diff
@@ -26,3 +30,40 @@ func TestNewAppsTransportKeyFromFile(t *testing.T) {
 		t.Fatal("unexpected error:", err)
 	}
 }
+
+type RoundTrip struct {
+	rt func(*http.Request) (*http.Response, error)
+}
+
+func (r RoundTrip) RoundTrip(req *http.Request) (*http.Response, error) {
+	return r.rt(req)
+}
+
+func TestAppsTransport(t *testing.T) {
+	customHeader := "my-header"
+	check := RoundTrip{
+		rt: func(req *http.Request) (*http.Response, error) {
+			h, ok := req.Header["Accept"]
+			if !ok {
+				t.Error("Header Accept not set")
+			}
+			want := []string{customHeader, acceptHeader}
```

Comment:
Where is customHeader being set in addition to "Accept"? I'm missing how this is done.